packages/teams-js/src/public/meeting.ts

```diff
@@ -210,6 +210,39 @@ export namespace meeting {
     error?: SdkError;
   }
 
+  /**
+   * Interface for mic state change
+   *
+   * @beta
+   */
+  export interface MicState {
+    id: number;
+    /**
+     * Indicates the mute status of the mic.
+     */
+    isMicMuted: boolean;
+  }
+
+  export interface MeetingFailedResponse {
+    id: number;
+    /**
+     * The reason for the failure
+     */
+    reason: string;
+    /**
+     * This property is currently unused.
```

Comment:
Why do you need an unused optional message property? That seems like it would overlap with the required reason property.